composer.json

```diff
@@ -32,40 +32,17 @@
         }
 
     ],
-    "require-dev": {
-        "doctrine/coding-standard": "^12.0.0",
-        "phpstan/phpstan": "^1.10.60",
-        "phpstan/phpstan-phpunit": "^1.3.16",
-        "phpunit/phpunit": "^10.5.12",
-        "vimeo/psalm": "5.23.0",
-        "roave/infection-static-analysis-plugin": "^1.34.0"
```

Comment:
This tool shares the same address space as the main library: moving it here may be a mistake (same for phpunit).

I think we should only move tools that operate completely independently, which currently includes phpcs, phpstan (not psalm, due to the coupling with the test suite mutations).

That said, I'm fine with trying this out anyway 👍